std/array.d

```diff
@@ -350,7 +344,7 @@ See_Also: $(REF Tuple, std,typecons)
  */
 
 auto assocArray(Range)(Range r)
-if (isInputRange!Range)
+    if (isInputRange!Range)
```

Comment:
@wilzbach we've gone back and forth with the style. I think `if` should be flush with the previous line (no extra indent). Could you please work your magic on the autotester to make it so? :)